checks/check_crl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# author: Oguzhan Cicek, OpenSource Security GmbH - oguzhan(at)os-s.de
```

Comment:
Please use our file headers:
```
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# Copyright (C) 2019 tribe29 GmbH - License: GNU General Public License v2
# This file is part of Checkmk (https://checkmk.com). It is subject to the terms and
# conditions defined in the file COPYING, which is part of this source code package.
```
In the past, I changed them while merging, but strictly speaking, you have to do this in order to declare that the code is under GPL v2.